src/planwise/component/projects.clj

```diff
@@ -49,10 +47,15 @@
 ;; ----------------------------------------------------------------------
 ;; Service functions
 
-(defn list-projects [service]
+(defn list-projects
+  [service]
```

Comment:
Yes we should. I've been using this style lately because it accomodates better to doc-strings. That and it's more uniform with the case of multiple arities.